tests/python_tests/helpers/device.py

```diff
@@ -117,21 +117,21 @@ def get_result_from_device(
     }
 
     # Handling "Bfp8_b" format separately with sfpu condition
-    if formats.pack_dst == DataFormat.Bfp8_b:
+    if formats.output_format() == DataFormat.Bfp8_b:
```

Comment:
```suggestion
    if formats.output_format == DataFormat.Bfp8_b:
```
Properties can be accessed without (). It looks more like accessing a field, not calling a function.